schema/2.0.0/seurat_encoding.md

```diff
@@ -0,0 +1,226 @@
+
+# Schema - Seurat encoding
+
+Contact: acarr@chanzuckerberg.com
+
+Document Status: _In progress_
+
+Schema version: 2.0.0
+
+
+## Overview
+
+All data submitted to the [cellxgene data portal](https://cellxgene.cziscience.com/) is automatically converted to a Seurat V3 object that can be loaded by the R package [Seurat](https://satijalab.org/seurat/). The conversion is done through [sceasy](https://github.com/cellgeni/sceasy).
+
+This document describes such Seurat encoding for the converted data and it mirrors the data [schema](./schema.md).
```

Comment:
This document describes the Seurat encoding for the converted data. Readers should be familiar with the [schema](.schema.md).